.travis.yml

```diff
@@ -4,7 +4,7 @@ node_js:
   - "0.12"
 
 before_install:
-  - gem install scss_lint -v 0.41.0 --no-rdoc --no-ri
+  - gem install scss_lint -v 0.42.2 --no-rdoc --no-ri
```

Comment:
Its like the color variables I had to fix with this update.
We can set this to newest but it can show random errors like colors mentioned before.